service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ReviewCMOControllerAboutToSubmitTest.java

```diff
@@ -0,0 +1,115 @@
+import static uk.gov.hmcts.reform.fpl.enums.HearingType.CASE_MANAGEMENT;
+import static uk.gov.hmcts.reform.fpl.model.order.CaseManagementOrder.sealFrom;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocumentReference;
+
+@ActiveProfiles("integration-test")
+@WebMvcTest(ReviewCMOController.class)
+@OverrideAutoConfiguration(enabled = true)
+public class ReviewCMOControllerAboutToSubmitTest extends AbstractControllerTest {
+
+    @MockBean
+    private DocumentSealingService documentSealingService;
+
+    protected ReviewCMOControllerAboutToSubmitTest() {
+        super("review-cmo");
+    }
+
+    @Test
+    void shouldSetReturnStatusAndRequestedChangesWhenJudgeRejectsOrder() {
+        CaseManagementOrder cmo = buildCMO();
+
+        CaseData caseData = CaseData.builder()
+            .draftUploadedCMOs(List.of(element(cmo)))
+            .reviewCMODecision(ReviewDecision.builder()
+                .changesRequestedByJudge("Please change XYZ")
+                .decision(JUDGE_REQUESTED_CHANGES).build())
+            .build();
+
+        CallbackRequest callbackRequest = CallbackRequest.builder().caseDetails(asCaseDetails(caseData)).build();
+
+        AboutToStartOrSubmitCallbackResponse response = postAboutToSubmitEvent(callbackRequest);
+
+        CaseData responseData = mapper.convertValue(response.getData(), CaseData.class);
+
+        CaseManagementOrder returnedCMO = responseData.getDraftUploadedCMOs().get(0).getValue();
+        assertThat(returnedCMO.getStatus()).isEqualTo(RETURNED);
+        assertThat(returnedCMO.getRequestedChanges()).isEqualTo("Please change XYZ");
+
+    }
+
+    @Test
+    void shouldSealPDFAndAddToSealedCMOsListWhenJudgeApprovesOrder() throws Exception {
+        DocumentReference sealedCMODocument = testDocumentReference();
+        given(documentSealingService.sealDocument(any(DocumentReference.class))).willReturn(sealedCMODocument);
+
+        UUID cmoId = UUID.fromString("51d02c7f-2a51-424b-b299-a90b98bb1774");
+        CaseManagementOrder cmo = buildCMO();
+
+        CaseData caseData = CaseData.builder()
+            .draftUploadedCMOs(List.of(element(cmoId, cmo)))
+            .hearingDetails(List.of(element(hearing(cmoId))))
+            .reviewCMODecision(ReviewDecision.builder().decision(SEND_TO_ALL_PARTIES).build()).build();
+
+        CallbackRequest callbackRequest = CallbackRequest.builder().caseDetails(asCaseDetails(caseData)).build();
+
+        AboutToStartOrSubmitCallbackResponse response = postAboutToSubmitEvent(callbackRequest);
+
+        CaseData responseData = mapper.convertValue(response.getData(), CaseData.class);
+
+        assertThat(responseData.getDraftUploadedCMOs()).isEmpty();
+        assertThat(responseData.getSealedCMOs()).isNotEmpty();
+        assertThat(responseData.getSealedCMOs().get(0).getValue()).isEqualTo(
+            sealFrom(sealedCMODocument, hearing(cmoId), LocalDate.now()));
```

Comment:
add 

verify(documentSealingService).sealDocument(cmo.getOrder());

or use:
given(documentSealingService.sealDocument(cmo.getOrder())).willReturn(sealedCMODocument);

to prove that correct document has been sealed